pydropsonde/helper/__init__.py

```diff
@@ -125,6 +125,70 @@
 l3_filename = "Level_3.nc"
 
 
+def get_chunks(ds, var):
+    dimensions = ds[var].dims
+    chunks = {
+        "sonde_id": min(256, ds.sonde_id.size),
+        "alt": min(350, ds.sonde_id.size),
+    }
+
+    return tuple((chunks[d] for d in dimensions))
+
+
+def add_encoding(ds, exceptions=[]):
+    variables = [
+        "u",
+        "v",
+        "ta",
+        "p",
+        "rh",
+        "lat",
+        "lon",
+        "gpsalt",
+        "alt",
+        "sonde_id",
+        "q",
+        "iwv",
+        "w_dir",
+        "w_spd",
+    ]
+
+    enc_var = {
+        var: {
+            "compression": "zstd",
+            "dtype": "float32",
+            "chunksizes": get_chunks(ds, var),
+        }
+        for var in variables
+        if var not in ds.dims
+        if var not in exceptions
+    }
+    enc_time = {
+        var: {
+            "compression": "zstd",
+            "chunksizes": get_chunks(ds, var),
+            "_FillValue": np.datetime64("NaT"),
```

Comment:
I haven't really, but it works like this and I thought maybe it's useful to use the time-nan value for time values. It felt a bit weird to just use some random huge float

Reply:
I agree that `NaT` is definitely a good option. Let's keep that and in case issues arise with the time encoding in the future, we can have a look into it again.